package.json

```diff
@@ -45,6 +45,7 @@
         "franc-min": "5.0.0",
         "interactjs": "1.10.0",
         "jquery": "3.5.1",
+        "js-video-url-parser": "^0.4.2",
```

Comment:
```suggestion
        "js-video-url-parser": "0.4.2",
```

Reply:
done in 7a3b0ff750a498c297ca6f45cf36cdd2ead77db9